OracleCoherence/README.md

```diff
@@ -0,0 +1,64 @@
+# Oracle Coherence 12c Docker Image
+
+This is a Dockerfile for [Oracle Coherence 12c](http://www.oracle.com/technetwork/middleware/coherence/overview/index.html). The purpose of this Docker container is to facilitate the setup of development and integration testing environments for developers.
+
+This image uses the Standalone Distribution of Coherence 12c.
+
+**IMPORTANT**: Oracle **does not support Docker** in any environment, including but not limited to Development, Integration, and Production environments.
+
+## Install Oracle Linux 7 Base Docker Image
+The Dockerfile contained in this project will build image based on the Oracle Linux 7.0 Base Docker Image. In order to easily install that, run the following script:
+
+	$ install-ol7-docker-image.sh
+
+For more information and documentation, read the [Docker Images from Oracle Linux](http://public-yum.oracle.com/docker-images) page.
```

Comment:
This line should be changed to point to https://registry.hub.docker.com/_/oraclelinux/